.pre-commit-config.yaml

```diff
@@ -1,6 +1,5 @@
 repos:
--   repo: https://gitlab.com/pycqa/flake8
-    rev: 3.8.4
-    hooks:
-    - id: flake8
-      additional_dependencies: ["flake8-tuple"]
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
It looks like the indentation is off, if I compare it with the examples from pre-commit's website

Reply:
Also, the same applies as in the other repo. We should still download flake8 (I also have the feeling that this syntax is not completely correct, looking at the docs).

Reply:
> It looks like the indentation is off, if I compare it with the examples from pre-commit's website

Yes, correcting in upcoming commit

Reply:
> We should still download flake8 (I also have the feeling that this syntax is not completely correct, looking at the docs, because you're using flake8 without specifying that the repo is local).

Sorry didn't quite got your message.  Can you please elaborate on `you're using flake8 without specifying that the repo is local`

Reply:
The [pre-commit docs](https://pre-commit.com) show examples of how to add hooks to a repo. The code above doesn't follow the guidelines in the docs. In order to be used, `flake8` needs to be installed. The installation happens thanks to the `repo` value, which should be the URL of the flake8 repository.
As written in the docs, `id` is the name of the hook that you want to use from the repository that you specified in `repo`. In this case, you're using the `flake8` hook from a repository that doesn't contain it.

Reply:
Regarding my comment about the `local` identifier: this is part of the advanced usage described in the docs. I think it's not necessary to use it here, but if you're curious you can check the docs [Repository local hooks](https://pre-commit.com/#repository-local-hooks).

Reply:
I see thanks, got you now. Please give this [commit](https://github.com/gridsingularity/myco-api-client/pull/24/commits/c9f374657fd61f1817a41e1bbbb91ca425ca37fa) a look.

Also created a PR for [d3a-interface](https://github.com/gridsingularity/d3a-interface/pull/286)